{"name": "test_add[bigint]", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('get_calc', <FixtureRequest for <Function test_add[bigint]>>)", "trace": "file D:\\pycharm\\HogwartsLG5\\test_pytest\\test_calc_new.py, line 9\n      def test_add(self, get_calc, get_datas):\nE       fixture 'get_calc' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, connectDB, doctest_namespace, get_datas, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, testrun_uid, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nD:\\pycharm\\HogwartsLG5\\test_pytest\\test_calc_new.py:9"}, "parameters": [{"name": "get_datas", "value": "[1000, 1000, 2000]"}], "start": 1610292898774, "stop": 1610292898774, "uuid": "87fe6433-a648-42a6-960d-e3eccc733647", "historyId": "985e0ef55ec79c83e07d71449373fd04", "testCaseId": "07d7074b11baf99ef0f511221e36f260", "fullName": "test_calc_new.TestCalc#test_add", "labels": [{"name": "suite", "value": "test_calc_new"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-B1515VS"}, {"name": "thread", "value": "15108-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_new"}]}